cluster/v1alpha1/types_addonplacementscore.go

```diff
@@ -58,6 +59,10 @@ type AddOnPlacementScoreItem struct {
 	// +kubebuilder:validation:Maximum:=100
 	// +required
 	Value int32 `json:"value"`
+
+	// Quantity is the original value of the score.
+	// +optional
+	Quantity *resource.Quantity `json:"quantity"`
```

Comment:
I am thinking to not adding quantity for now. I am still concerned that too frequent update on this and potential overuse.

Reply:
yes, potential overuse would be an issue, remove this from PR.